pkg/analysis_server/test/services/correction/fix_test.dart

```diff
@@ -5986,6 +5986,74 @@ var x;
 ''');
   }
 
+  test_removeMethodDeclaration_getter() async {
+    String src = '''
+class A {
+  int x;
+}
+class B extends A {
+@override
+int get /*LINT*/x => super.x;
+}
+''';
+    await findLint(src, LintNames.unnecessary_override);
+
+    await applyFix(DartFixKind.REMOVE_METHOD_DECLARATION);
+
+    verifyResult('''
+class A {
+  int x;
+}
+class B extends A {
+
+}
+''');
+  }
+
+  test_removeMethodDeclaration_method() async {
+    String src = '''
+class A {
+@override
```

Comment:
@bwilkerson is there a way to safely remove the indentation and the new line when removing the method?